qemu/tests/cfg/memhp_threads.cfg

```diff
@@ -0,0 +1,30 @@
+- memhp_threads:
+    type = memhp_threads
+    required_qemu = [5.0, )
+    paused_after_start_vm = yes
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    smp = 4
+    guest_numa_nodes = "node0"
+    mem_devs = mem0
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 4096M
+    backend_mem_mem0 = memory-backend-ram
+    numa_memdev_node0 = mem-mem0
+    no Host_RHEL.6
+    no RHEL.5
+    no Windows..i386
+    no WinXP Win2000 Win2003 WinVista
+    ppc64,ppc64le:
+        threshold = 0.15
```

Comment:
I did not see any call, could you explain this ? Thanks!

Reply:
It's used in check_memory() in virttest/utils_test/qemu/__init__.py.

Reply:
Got it, thanks!